src/main/java/task/Event.java

```diff
@@ -0,0 +1,59 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an Event task in the chatbot system. An Event object contains a
+ * description, done status, 'from' date, and 'to' date.
+ */
+public class Event extends Task {
+    protected LocalDate from;
+    protected LocalDate to;
```

Comment:
Perhaps the variable names can be changed to something that has a clearer explanation?
```suggestion
    protected LocalDate fromDate;
    protected LocalDate toDate;
```